Docker/Exe_5/HelloWorld/app.js

```diff
@@ -0,0 +1,41 @@
+var createError = require('http-errors');
```

Comment:
Simplify this code, we just need a node server running, also remove the `node_modules`